### ISSUE TITLE
Making the script working both on messenger.com and facebook.com/messages/t

### ISSUE THREAD

**Issue by Eminmpenza** [abdallah-eldali/fire-messenger#2]
Hi,

Could it be possible to make this awesome script also working on facebook.com/messages/t ?

I rather use this way and URL because I find it way less slow and bloated than messenger.com (like a messenger lite on desktop) when it comes to browsing messages.

Maybe it could speed up the process or allow it to be smoother.

Also, sometimes I'm not even able to access messenger.com (it won't load for some reason) and I can only use facebook.com/messages/t

Thanks in advance for your great work !

**Comment by abdallah-eldali** [comment]
Hello, thank you for creating an issue ticket (and for being the first one!).

I'll look into this issue and most likely ship alongside this ongoing [PR](https://github.com/abdallah-eldali/fire-messenger/pull/1)

**Comment by abdallah-eldali** [comment]
The issue has already been completed. 

A note for everyone reading this: Due to the nature of this project, there might some use cases where the extension breaks when using it on `facebook.com/messages` that I might have not caught while testing it. Thus, I would greatly appreciate if any user finds a bug, to please report it by creating an issue ticket.

Also, while testing I personally didn't find any performance improvements when using the extension on `facebook.com/messages` vs. on `messenger.com`. Personally I feel both websites are bloated and causes slowdowns (this is due to Meta, not the extension itself).

The update can be found in GitHub's release page. I'll deploy it on the [AMO](addons.mozilla.org) soon after completing a more exhaustive testing.

EDIT: Typo

**Comment by Eminmpenza** [comment]
Thank you so much for the update.

It's super super fast now ! I tested it on both and as you said it's the same speed, but it's not slow and bloated as it used to feel before though.

The process crashes eventually at some point, so we need to keep an eye on it and relaunch when necessary (see attached screenshoot)

![image](https://github.com/user-attachments/assets/1075e447-bbca-4d94-8378-c42fe5e77626)

But it's not a big deal since I used to stop and restart it myself a many times in the previous release because it was buggy and it was skipping messages.
This new update doesn't do that anymore. No messages are skipped or avoided. 
It gets the job done so efficiently and fast !

This update is a big improvement and a major step forward, thanks again !
